INSTALL

```diff
@@ -169,6 +169,27 @@ Some building considerations
 
      bash glvis-keygen.sh ["Your Name"] ["Your Email"]
 
+- On some versions of Mac OS X, an error may be encountered with GLVis running
```

Comment:
I think you mean: `Mac OS X` --> `macOS`.

Reply:
We actually have `Mac OS X` in this file and once in `CHANGELOG`.

Should we just replace all of them with `macOS`?

Reply:
> Should we just replace all of them with `macOS`?

It probably makes sense -- according to the Wikipedia page, the last version to use the name `Mac OS X` was `Lion`/10.7, then the name was just `OS X` till `El Capitan`/10.11 (released 2015), and since `Sierra`/10.12 (released 2016) it is `macOS`.

In any case, people will know what we're talking about even if we don't change it to `macOS` everywhere. 😁

Reply:
Lets keep `Mac OS X` for now. We can update it both here and in mfem at a later time.